### PR TITLE
Make header sticky, combine AppHeader and ProposalFilter

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: imports/core/startup/client/routes.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import VotingPage from '../../../voting/ui/pages/VotingPage.vue';
```
Comment (ducdev) [inline imports/core/startup/client/routes.js:1]: It seems you forgot to make change for `imports/voting/index.js`
```
export { VotingPage } from './ui/pages/VotingPage.vue';
```

File: imports/core/ui/components/AppHeader.vue [hunk 2]
```diff
@@ -1,61 +1,125 @@
...
   @import "/imports/core/ui/styles/variables";
 
-  header {
-    position: relative;
-    flex-grow: 0;
-    flex-shrink: 0;
-    background: $brand-color;
-    @media screen and (max-width: $container-width){
-      padding: 10px;
+  .hidden, .fixed {
```
Comment (ducdev) [inline imports/core/ui/components/AppHeader.vue:54]: image
The header is now covering the first proposal item in list.

**Review by ducdev — CHANGES_REQUESTED** [review]
Please do some quick fixes @davidyuk

File: imports/core/ui/components/AppHeader.vue (hunk 2 above)
Comment (davidyuk) [reply imports/core/ui/components/AppHeader.vue:54]: I can't reproduce (Chromium 61, Ubuntu).

**Pull request merged**